### PR TITLE
add licensing page

### LINKED ISSUE #3875: Add bundle license file
## 🧰 Task
@jaimergp @goanpeca @jni

I believe we should add a specific "bundle" license file to the repo that is LGPL if we ship pyside2 or GPL if we ship pyqt5 in the bundle. This will then appear in the installer that @jaimergp is working on. Right now it looks like 

image

which I believe we need to change before we distribute the bundle

### LINKED ISSUE #4095: License info for limited release of bundled app
### Context
The napari bundled app will have a limited release in March that will allow users to install napari viewer along with the majority of available plugins (final number of plugins during initial release TBD). As part of the initial release, license information will need to be made available to users.

### Goal
Surface licensing information in a location accessible to users (e.g., napari.org)
- [X] generate output of script
- [x] store license list in docs (location to live under "Community" at the same level as "Roadmaps" and "Releases")
- [x] surface license to users in napari install process (e.g., via link)

### PR DESCRIPTION
**Pull request by jaimergp**
# Description

## Type of change

- [ ] Bug-fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] This change requires a documentation update

# References

Closes #4095 , closes #3875 

# How has this been tested?

- [ ] example: the test suite for my feature covers cases x, y, and z
- [ ] example: all tests pass with my change

## Final checklist:
- [ ] My PR is the minimum possible work for the desired functionality
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If I included new strings, I have used `trans.` to make them localizable.
      For more information see our [translations guide](https://napari.org/docs/dev/guides/translations.html).

### CODE REVIEW

**Comment by jaimergp** [comment]
Questions:

- Check the TOC location is correct
- Check we like the URL; right now it'd be https://napari.org/community/licensing.html, which might be misleading? Whatever we choose, it needs to stay for as long as possible, since old installers will still refer to this location. IOW, we have to get it right now.

File: docs/community/licensing.md [hunk 1]
```diff
@@ -1,941 +1,10 @@
 # Licensing
 
 napari is distributed under the BSD-3-Clause license, a copy of which is available
-[here][repo_license].
+[here][https://github.com/napari/napari/blob/latest/LICENSE].
```
Comment (melissawm) [inline docs/community/licensing.md]:
```suggestion
[here](https://github.com/napari/napari/blob/latest/LICENSE).
```

File: docs/community/licensing.md [hunk 2]
```diff
@@ -1,941 +1,10 @@
...
 
+```{include} ../../LICENSE
 ```
```
Comment (melissawm) [inline docs/community/licensing.md]: For consistency, might be worth doing

````suggestion
```{include} ../../LICENSE
:literal: true
```
````

**Review by melissawm — COMMENTED** [review]
Left a couple suggestions, thanks @jaimergp

File: docs/community/licensing.md (hunk 2 above)
Comment (jaimergp) [reply docs/community/licensing.md]: Done! (on both includes) I am curious what's the difference? Or maybe I shouldn't have added the literal on the 2nd one?

**Comment by liu-ziyang** [comment]
@jaimergp can you please take a look at the failing test? if there is no other things we need to update I would like us to freeze change and rally revieweres

**Comment by jaimergp** [comment]
Done!

**Review by melissawm — APPROVED** [review]
LGTM, thanks!

**Pull request merged**